src/Controller/Api/GalleryController.php

```diff
@@ -216,7 +217,10 @@ public function getGalleryMediasAction($id)
 
         $media = [];
         foreach ($galleryItems as $galleryItem) {
-            $media[] = $galleryItem->getMedia();
+            $itemMedia = $galleryItem->getMedia();
```

Comment:
Maybe we could use `$medium` instead of `$itemMedia` here.

Reply:
Fixed